contrib/networkmanager-openconnect/update.py

```diff
@@ -0,0 +1,2 @@
+url = "https://gitlab.gnome.org/GNOME/NetworkManager-openconnect/-/tags"
+pattern = r"href=\"/GNOME/NetworkManager-openconnect/-/tags/([0-9.]+)\""
```

Comment:
```suggestion
pattern = r"NetworkManager-openconnect-([0-9.]+).tar.gz"
```

you can do something similar in the other mr too, never need the whole href thing